geode-redis/src/acceptanceTest/java/org/apache/geode/redis/internal/executor/pubsub/PubSubNativeRedisAcceptanceTest.java

```diff
@@ -16,9 +16,11 @@
 package org.apache.geode.redis.internal.executor.pubsub;
 
 import org.junit.ClassRule;
+import org.junit.Ignore;
 
 import org.apache.geode.NativeRedisTestRule;
 
+@Ignore
```

Comment:
I'd suggest having a geode ticket that will not be fixed until this Ignore can be removed. You can reference that ticket in the Ignore comment. I think it is okay for that to be GEODE-8577. You just need to be careful to not mark GEODE-8577 as fixed when you push this ignore.

Reply:
Done.